modules/index.js

```diff
@@ -0,0 +1,118 @@
+/* eslint max-classes-per-file: ["error", 3] */
+import Books from './book.js';
+import Store from './store.js';
+import { DateTime } from './luxon.js';
+
+class task {
+  static currentBooks() {
+    const books = Store.getBooks();
+    books.forEach((book) => task.addBookList(book));
+  }
+
+  static addBookList(book) {
+    const list = document.querySelector('#list-book');
+    const row = document.createElement('tr');
+    row.innerHTML = `<td>${book.title}</td>
+      <td>${book.author}</td>
+      <td><a href="#" class="btn btn-danger btn-sm delete">REMOVE</a></td>`;
+    list.appendChild(row);
+  }
+
+  static deleteBook(el) {
+    if (el.classList.contains('delete')) {
+      el.parentElement.parentElement.remove();
+    }
+  }
+
+  static showAlert(message, className) {
+    const div = document.createElement('div');
+    div.className = `alert alert-${className}`;
+    div.appendChild(document.createTextNode(message));
+    const container = document.querySelector('.firstContainer');
+    const navigation = document.querySelector('.navigation');
+    container.insertBefore(div, navigation);
+    setTimeout(() => document.querySelector('.alert').remove(), 3000);
+  }
+
+  static clearField() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
+}
+
+const listBook = document.getElementById('listBooks');
+const newBook = document.getElementById('newBooks');
+const contact = document.getElementById('contact');
+const navItems = [listBook, newBook, contact];
+const listSection = document.getElementById('list-section');
+const addSection = document.getElementById('add-section');
+const contactSection = document.getElementById('contact-section');
+const sections = [listSection, addSection, contactSection];
+
+function saveActiveNavItemLocally(id) {
+  localStorage.setItem('activeNavItem', id);
+}
```

Comment:
- [ ] in this milestone you are expected to use ES6 which includes the usage of arrow functions over the `function` keyword.

Reply:
hello @mikethreels, come again on reloading the page because on my side it's functioning well, when I delete an item and then reload my page I can't see the undefined items